mypy/util.py

```diff
@@ -906,3 +919,20 @@ def quote_docstring(docstr: str) -> str:
         return f"''{docstr_repr}''"
     else:
         return f'""{docstr_repr}""'
+
+
+_default = object()
+
+
+@overload
+def getattr(value: object, name: str) -> Any: ...
+@overload
+def getattr(value: object, name: str, default: T) -> Any | T: ...
+def getattr(value, name, default: object = _default):
+    if default is _default:
+        return builtins_getattr(value, name)
+    return builtins_getattr(value, name, default)
+
+
+def attrgetter(name: str) -> operator.attrgetter[Any]:
+    return operator.attrgetter(name)
```

Comment:
why?

Reply:
so I updated `operator.getattr` to not be any, but there's several usages in mypy that I want to avoid reworking, for the sake of avoiding conflicts with upstream. so frustratingly I think this is the most minimal solution

Reply:
it sucks that there's no `AttrOf`...

Reply:
why can't we make one?

Reply:
I'm guessing you already have an issue for it?

Reply:
not sure if this ones qualifies
- #668

Reply:
That's actually a very cool way to define inline protocols 😮

Reply:
But yea, it's not as dynamic as an `AttrOf[T, NameT]` or `Attr[NameT][T]`, where `NameT: str` could be too.

Reply:
#800